rcgen/src/crl.rs

```diff
@@ -5,12 +5,13 @@ use time::OffsetDateTime;
 use yasna::DERWriter;
 use yasna::Tag;
 
+use crate::CertificateParams;
```

Comment:
nit: I think this should be in the `use crate::{...}` group on L11..15.

Reply:
Fixed.